src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ctaLinkIcons, ctaLinkDisplay, ctaLinkColors, ctaLinkAnimations, ctaLinkButtonSizes, ctaLinkButtonVariants } from './CtaLink.levers';
+import Icon from 'react-hero-icon';
+import clsxd from 'clsx-dedupe';
+
+/**
+ * CTA Link Component
+ *
+ */
+export const CtaLink = ({ classes = {}, text, srText, display, link, color, isButton, variant, size, icon, animate, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // display
+  if (display && ctaLinkDisplay.includes(display)) {
+    switch (display) {
+      case 'inline-block':
+        levers.display = 'su-inline-block';
+        break;
+
+      case 'block':
+        levers.display = 'su-block';
+        break;
+
+      case 'flex':
+        levers.display = 'su-flex';
+        break;
+    }
+  }
+
+  // color
+  if (color && ctaLinkColors.includes(color) && !isButton) {
```

Comment:
Why not allow for buttons to have color?

Reply:
This is just the color of the text. We tend to not change button text color much (since it's either white, or digital red - for the outline one, or totally custom by passing in classes).